docs/upgrade-guide.md

```diff
@@ -62,7 +62,7 @@ v7 is a complete rewrite of the library. It is redesigned to be fast, lightweigh
 
 - `MapContext` is now an official API. The experimental `_MapContext` export will be removed in a future release.
 - `react-virtualized-auto-sizer` is no longer a dependency.
-- Inertia has been enabled by default on the map controller. To revert to the behavior in previous versions, set the [interaction options](/docs/api-reference/interactive-map.md#interaction-options):
+- Inertia has been enabled by default on the map controller. To revert to the behavior in previous versions, set the [interaction options](https://github.com/visgl/react-map-gl/tree/6.1-release/docs/api-reference/interactive-map.md#interaction-options):
```

Comment:
Is the idea that older deprecated pages are available as markdown, but no longer needed on the website?

Reply:
Docusaurus fails to build if you have invalid links. In this case, the links were written according to the directory structure of an earlier release, and the files no longer exist.

Reply:
Ah, ok good to know. Sounds like we could either link to the repo in all what's new and upgrade guides going forward, or let the build tool tell us when it's time to switch it from the nicer website experience to the repo.